Mathlib/Topology/Category/TopCat/Sphere.lean

```diff
@@ -20,20 +20,35 @@ universe u
 
 namespace TopCat
 
-/-- The `n`-sphere is the set of points in ℝⁿ⁺¹ whose norm equals `1`,
-endowed with the subspace topology. -/
-noncomputable def sphere (n : ℤ) : TopCat.{u} :=
-  TopCat.of <| ULift <| Metric.sphere (0 : EuclideanSpace ℝ <| Fin <| (n + 1).toNat) 1
-
 /-- The `n`-disk is the set of points in ℝⁿ whose norm is at most `1`,
 endowed with the subspace topology. -/
 noncomputable def disk (n : ℤ) : TopCat.{u} :=
   TopCat.of <| ULift <| Metric.closedBall (0 : EuclideanSpace ℝ <| Fin <| n.toNat) 1
```

Comment:
By writing `𝕊 (-1)` as `∂𝔻 0`, I think it is now safe to get rid of `ℤ` altogether. Thanks for your refactor.

Also, please remove the doc-string on lines 14 and 15 that explains the use of `ℤ`.

```suggestion
noncomputable def disk (n : ℕ) : TopCat.{u} :=
  TopCat.of <| ULift <| Metric.closedBall (0 : EuclideanSpace ℝ (Fin n)) 1
```